source/states/loading_state.h

```diff
@@ -20,8 +19,6 @@ class loading_state : public state {
 	int m_bar_scale;
 
 	Stardust::Utilities::Timer* m_animation_timer;
-	Stardust::Graphics::UI::UIText* loading_what_text;
-	Stardust::Graphics::UI::UIText* input_text;
```

Comment:
we need a declaration for this text why did you remove it?

Reply:
UI Text does not exist.